glommio/src/io/bulk_io.rs

```diff
@@ -14,6 +14,51 @@ use std::{
     rc::Rc,
 };
 
+/// Set a limit to the size of merged IO requests.
+#[derive(Debug)]
```

Comment:
we should probably implement Default for this. I think `DeviceMaxSingleRequest` is a good default.

Reply:
I can't implement `Default` because the value of `DeviceMaxSingleRequest` depends on the major/minor device ids

Reply:
what do you mean? 

The actual value changes but all you're doing in this enum is to pick the variant. You don't need to fill in the value.

Reply:
Indeed 🤦‍♂️